packages/central-server/src/apiV2/surveyResponses/saveResponsesToDatabase.js

```diff
@@ -66,7 +66,10 @@ function buildResponseRecord(user, entitiesByCode, body) {
     approval_status: approvalStatus,
   } = body;
 
-  const timezoneName = getTimezoneNameFromTimestamp(timestamp);
+  const additionalConfigs = {};
+  if (timestamp) {
+    additionalConfigs.timezone = getTimezoneNameFromTimestamp(timestamp);
+  }
```

Comment:
Is this change so that it works even if we can't get the timezone name from the timestamp?

Reply:
Indeed, `timezone` needs to be passed in from meditrak-app. Thanks for catching that.